docs/network-policy.md

```diff
@@ -0,0 +1,361 @@
+# NetworkPolicy
```

Comment:
nit: feel free to ignore this comment but sometimes you use "network policy" and other times "NetworkPolicy" should it be consistently one of these?

Reply:
Good point!